matlab/MeCorr/MATLAB_CODE_me.m

```diff
@@ -0,0 +1,318 @@
+%                 model2.c(rxnpos) = epsilon_methylation ;
+%                   [fluxstate_gurobi] =  constrain_flux_regulation(model2,onreactions1h,offreactions1h,kappa,rho,epsilon,MODE,epsilon2,minfluxflag);
+%                  grate_ccle_exp_soft_hdacsign(i,kk + 4) = fluxstate_gurobi(rxnpos); %acetylation flux
+%                 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%             end
+%         end
+
+    else
+        grate_ccle_exp_soft(i,:) = NaN;
+%        grate_ccle_exp_soft_hdacsign(i,:) = NaN;
+    end
+end
+
+
+figure; h = histogram(grate_ccle_exp_soft(:,2),70);
+ xlabel('Predicted methylation flux')
+ ylabel('Total cell lines')
+title('Distribution of methylation flux among CCLE cell lines','fontweight','normal')
+
+save('VariablesSaved\grate_ccle_exp_soft_1E-4', 'grate_ccle_exp_soft');
+save('VariablesSaved\fluxstate_gurobi_1E-4', 'fluxstate_gurobi');
+save('VariablesSaved\solverobj_ccle_1E-4', 'solverobj_ccle');
+%% predicting sensitivity to hme inhibitors based on basal metabolic state - comparison with drug sensitivity data from seashore-ludlow study
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+hmei_list = {'BRD-A02303741';'BIX-01294';'methylstat';'QW-BI-011';...
+    'UNC0321';'CBB-1007';'UNC0638';'GSK-J4'};
+hmei_list= cell2table(hmei_list);
+hmei_list.Properties.VariableNames{'hmei_list'}='cpd_name';
+clearvars r_basal
+r_basal(height(hmei_list))= struct('rho',0, 'p',0, 'rlowerbound',0, 'rupperbound',0);
+pp_basal= zeros(height(hmei_list), 3);
+for j = 1:height(hmei_list)
+    fx = find(ismember(ctd2compoundidname_name, hmei_list(j,1)));
+    ix = ismember(drug_auc_expt(:,3), ctd2compoundidname_id(fx,1));
+    sum(ix) % 847. ix is a 1D array. Each drug appears multiple times (appears for each medium in drug_auc_expt)
+    hme_auc_dat = drug_auc_expt(ix,:);
+    
+    [ix, pos] = ismember(hme_auc_dat(:,1), exptidcelllinemediamatch(:,1)); sum(ix); %
+    v1 = hme_auc_dat(:,2); v1= table2array(v1);
+    v2 = grate_ccle_exp_soft(pos,:); % growth rate
+    v3 = fluxstate_gurobi(pos,:);
+    
+    % Scatter plots of methylation (metabolic) flux & calculate correlation
+    figure;
+    scatter(v1, v3)
+    ylabel({'Methylation Flux'},'fontname','helvetica'); %'fontweight','bold')
+    xlabel({'Sensitivity (AUC)'; hmei_list.cpd_name{j}},'fontname','helvetica');
+    ylim([0, 0.06])
+    [R, P, RL, RU]= corrcoef(v1, v3);
+    r_basal(j).rho=R; r_basal(j).p=P; r_basal(j).rlowerbound=RL; r_basal(j).rupperbound=RU;
+    str=sprintf('r= %1.3f',r_basal(j).rho(1,2));
+    T = text(min(get(gca, 'xlim')), max(get(gca, 'ylim')), str);
+    set(T, 'fontsize', 14, 'verticalAlignment', 'top', 'horizontalAlignment', 'left');
+    
+    % Boxplot pairs of methylation flux (Actually grapphing metabolic flux, 
+    ...but we hypothesize met flux approximates methyl flux)
+    ix0 = (v3 < 0.05);sum(ix0); % 0.05 is  the default threshold value

```

Comment:
How did you set the low flux threshold to be 0.05? This value should be the mean or median value.